roles/conjur_host_identity/tests/test.sh

```diff
@@ -133,7 +136,7 @@ function main() {
   ansible_cid=$(docker-compose ps -q ansible)
 
   run_test_cases
-  rm -rf compose_project_name
+
```

Comment:
nit: you've added a few empty lines @ 83, 119 and 139. We could remove them to clean up the file a bit.